pynamodb/models.py

```diff
@@ -169,40 +171,104 @@ class MetaModel(AttributeContainerMeta):
     """
     def __init__(cls, name, bases, attrs):
         super(MetaModel, cls).__init__(name, bases, attrs)
+
+        # We don't' allow inheritance from multiple direct parents
+        if (len(bases) > 1):
+            raise InheritanceError("Model can't inherit from multiple direct parents")
+
```

Comment:
This isn't a realistic restriction we can place on all classes that extend Model.
As far as I can tell this would break anyone using Mixins: e.g. `class MyModel(Model, Loggable)`